README.md

```diff
@@ -145,3 +145,41 @@ Privacy information can be found at https://privacy.microsoft.com/en-us/
 
 Microsoft and any contributors reserve all other rights, whether under their respective copyrights, patents,
 or trademarks, whether by implication, estoppel or otherwise.
+
+
+## Citation  
+[AutoGen](https://arxiv.org/abs/2308.08155). 
+AND  [EcoOptiGen](https://arxiv.org/abs/2303.04673).
+``` 
+bibtex
```

Comment:
also remove the 'bibtex' word, which is not necessary

Reply:
@JieyuZ2  okay done with changes :: review